python/examples/math/program.py

```diff
@@ -0,0 +1,165 @@
+from __future__ import annotations
+import asyncio
+import json
+from textwrap import dedent, indent
+from typing import TypeVar, Any, Callable, Awaitable, TypedDict, Annotated,  NotRequired, override, Sequence
+
+from typechat import (
+    Failure,
+    Result,
+    Success,
+    TypeChatModel,
+    TypeChatValidator,
+    TypeChatTranslator,
+    python_type_to_typescript_schema,
+)
+import collections.abc
+
+T = TypeVar("T", covariant=True)
+
+def Doc(s: str) -> str:
+    return s
+
+
+program_schema_text = '''
```

Comment:
Why not just generate this with `python_type_to_typescript_schema` so you have a single source of truth?